src/sidebar/directive/annotation-share-dialog.js

```diff
@@ -4,73 +4,69 @@ var angular = require('angular');
 
 var scopeTimeout = require('../util/scope-timeout');
 
-module.exports = function () {
-  return {
-    bindToController: true,
-    controllerAs: 'vm',
-    restrict: 'E',
-    template: require('../templates/annotation_share_dialog.html'),
-    // @ngInject
-    controller: function ($scope, $element) {
-      var shareLinkInput = $element.find('input')[0];
+module.exports = {
+  controllerAs: 'vm',
+  template: require('../templates/annotation_share_dialog.html'),
+  // @ngInject
+  controller: function ($scope, $element) {
```

Comment:
It might also be nice in follow up work to define all controllers as `AnnotationShareDialogController` functions at the top of the files, and just reference them in the `exports` instead of doing them inline. I think it makes the code a bit tidier and easier to read, and might also make debugging / error messages better. (Some of the components already do this.)